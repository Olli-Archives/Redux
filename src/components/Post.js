import React from 'react';
import PropTypes from 'prop-types';

function Post({ title }){
  return (
    <p>{title}</p>
  );
}

Post.propTypes = {
  title:PropTypes.string.isRequired
};

export default Post;

