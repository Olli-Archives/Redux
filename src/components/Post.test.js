import React from 'react';
import Post from './Post';
import { shallow } from 'enzyme';

describe('Post component', ()=>{
  it('renders a title', ()=>{
    const wrapper = shallow(<Post title='test title'/>);
    expect(wrapper).toMatchSnapshot();
  });
})
;
