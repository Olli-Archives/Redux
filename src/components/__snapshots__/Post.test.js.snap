// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post component renders a title 1`] = `
<p>
  test title
</p>
`;
